pkgs/development/haskell-modules/hoogle.nix

```diff
@@ -23,12 +23,16 @@
 # This will build mmorph and monadControl, and have the hoogle installation
 # refer to their documentation via symlink so they are not garbage collected.
 
-{ lib, stdenv, buildPackages
-, hoogle, writeText, ghc
-, packages
+{ lib, stdenv, buildPackages, haskellPackages
+, writeText
 }:
 
+selectPackages:
```

Comment:
```suggestion
# This argument is a package selector for Haskell packages.
# Example:
#   (hpkgs: [ hpkgs.mtl hpkgs.lens ])
selectPackages:
```

I think it is obvious to us what this `selectPackages` argument is, but I imagine it might not be as obvious to a new user.